examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -77,8 +79,12 @@
 )
 
 categorical_features = ["embarked", "sex", "pclass"]
-categorical_transformer = OneHotEncoder(handle_unknown="ignore")
-
+categorical_transformer = Pipeline(
+    steps=[
+        ("encoder", OneHotEncoder(handle_unknown="ignore")),
+        ("selecter", SelectPercentile(chi2, percentile=50)),
```

Comment:
I see this is included to add `chi2` to an example, but it does increase the example's runtime from 2 seconds to 13 seconds on the CI. I am okay with:

- Removing it all together or leaving it at `percentile=50`
- Not search through it in the grid search.
- Search through less `percentile` values.

Reply:
I did some other changes. I used a `RandomizedSearchCV` and limit the number of iterations and add some narrative to explain `n_iter` and the `GridSearchCV`. I reduced `cv=10` to the default `cv=5`.